src/components/button-toggle/button-toggle.component.js

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Icon from '../icon';
+import { StyledButtonToogle, StyledButtonToogleLabel, StyledButtonToggleIcon } from './button-toggle.style';
+import guid from '../../utils/helpers/guid';
+
+const ButtonToggle = (props) => {
+  const inputGuid = guid();
+
+  function icon() {
```

Comment:
could you change that name to be more meaningful? e.g.: `renderIcon`.

Reply:
I think it's fairly clear what this method is doing.